README.md

```diff
@@ -183,9 +195,11 @@ validation report. There is also a `check()` method that returns an Exception
 only with the first error found:
 
 ```php
+import Respect\Validation\Exceptions\ValidatorExceptionInterface
+
 try {
     $usernameValidator->check('really messed up screen#name');
-} catch(\InvalidArgumentException $e) {
+} catch(ValidatorExceptionInterface $e) {
     echo $e->getMainMessage();
```

Comment:
You are right.  I tried: 

```php
try {
    $usernameValidator = v::alnum()->noWhitespace()->length(1,15);
    $usernameValidator->check('really messed up screen#name');
} catch (\Exception $e) {
    var_dump(get_class($e));
    var_dump($e->getMainMessage());
    var_dump($e->getFullMessage());
}
```
When `$e->getFullMessage()` is called **it causes an undefined method fatal error**.